test/event.test.js

```diff
@@ -3,13 +3,15 @@
 const { test } = require('tap')
 const { join } = require('path')
 const ThreadStream = require('..')
+require('why-is-node-running')
```

Comment:
I think a separate issue should be opened and worked on to investigate why this is "fixing" the test suite.

Reply:
I think there is some weird interaction between tap own use of `async_hooks`, `worker_threads` and code coverage.

Reply:
It's affecting the Pino tests as well. We get flakes over there all the time.